### PR TITLE
upd(alacritty): `0.10.0` -> `0.10.1`

### PR DESCRIPTION
**Pull request by warofzen**

### CODE REVIEW

**Review by wizard-28 — COMMENTED** [review]
@kaistian do you wish to continue maintaining this pacscript?

**Comment by wizard-28** [comment]
@Henryws the compilation failed due to a `rustc` error.

**Review by wizard-28 — COMMENTED** [review]
@warofzen do you wish to maintain this pacscript?

**Comment by warofzen** [comment]
> @warofzen do you wish to maintain this pacscript?

Yeah, sure.

**Comment by Henryws** [comment]
> @Henryws the compilation failed due to a `rustc` error.

Narrowed it down to a cargo version issue. Installing with rustup works fine on debian stable, but the current ubuntu docker works fine ootb.

**Comment by wizard-28** [comment]
> > @Henryws the compilation failed due to a `rustc` error.
> 
> Narrowed it down to a cargo version issue. Installing with rustup works fine on debian stable, and the current ubuntu docker works fine ootb.

Yeah... so what to do here?

**Comment by Henryws** [comment]
As I've said on discord, if it doesn't work on debian but it does on Ubuntu, we should merge, as we technically only support Ubuntu, and debian support is taken with a grain of salt.

File: packages/alacritty/alacritty.pacscript [hunk 1]
```diff
@@ -1,9 +1,10 @@
 name="alacritty"
```
Comment (wizard-28) [inline packages/alacritty/alacritty.pacscript:1]:
```suggestion
if [[ "$(grep -Po 'ID=\K[^"]+' /etc/os-release)" == "debian" ]]; then
    fancy_message error "This script doesn't work on Debian"
    cleanup # Trigger pacstall cleanups
    exit 1
fi

name="alacritty"
```

Block installation on Debian.

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Review by wizard-28 — APPROVED** [review]

**Pull request merged**